libs/s25client/s25client.cpp

```diff
@@ -296,7 +298,7 @@ bool InitDirectories()
 #endif
     if(!oldSettingsDir.empty() && bfs::is_directory(oldSettingsDir))
     {
-        const std::string newSettingsDir = RTTRCONFIG.ExpandPath(FILE_PATHS[94]);
+        const std::string newSettingsDir = RTTRCONFIG.ExpandPath(s25::folders::config);
         if(bfs::exists(newSettingsDir))
         {
             s25util::error(std::string("Old and new settings directory found. Please delete the one you don't want to keep!\nOld: ")
```

Comment:
This text should be translateable (but separate issue)

Reply:
I Implemented a change to this in https://github.com/Return-To-The-Roots/s25client/pull/1254/commits/45104bf8cf7b3f32c506a33387fe4b23aa3c2299

As the code executes before anything is loaded the string must not be translatable and LOG must not be used yet.

The new code does this and is nicely extendable if we chose to do the renaming proposed in Discord